interview.md

```diff
@@ -58,11 +86,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+- It would be 100% language design since all implementation will already be done, we are just trying to improve user experience. 
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+- Very big, depending on the language chosen. 
+- 
 ### Benefits and drawbacks
 
 _Why might this be a good idea for a project? Why might this not be a good idea
 project?_
+
+- pros: very practical, very applied 
+- cons: not well defined, only improvement, very ambitions. 
```

Comment:
I agree. This is a great idea for a DSL and would probably actually be used, but it seems like it would be difficult for one student to complete in less than a semester.